modules/ROOT/pages/_partials/generic-provisioning.adoc

```diff
@@ -1,6 +1,7 @@
 To provision the {service-name} mobile service:
 
 . Log into the OpenShift console.
+. Choose your project.
```

Comment:
doesn't it default to a project?

Reply:
It just brings you to catalog page and you have to choose your project from the list on the side navbar with list of projects.

to be able to choose "Add to project" we need to select the project first